source/sgp_mode/ModularityAnalysis.h

```diff
@@ -46,8 +46,8 @@
 emp::vector<int> GetUsefulStarts(emp::vector<emp::vector<int>> task_programs) {
   emp::vector<int> list_of_starts = {};
 
-  for (int y = 0; y < task_programs.size(); y++) {
-    for (int e = 0; e <= task_programs[y].size() - 1; e++) {
+  for (int y = 0; y < (int)task_programs.size(); y++) {
```

Comment:
same for these for loops, would be better as 'size_t' and that will probably remove the need to cast to an int